panopticum/models.py

```diff
@@ -223,7 +223,7 @@ def __str__(self):
         return "%s" % self.name
 
 
-class ComponentRuntimeTypeModel(models.Model):
+class ComponentTypeModel(models.Model):
```

Comment:
TIme to return back migrations to git. Because we have released instance.
Relative [SO topic](https://stackoverflow.com/questions/28035119/should-i-be-adding-the-django-migration-files-in-the-gitignore-file) about migrations and collaborations.

Reply:
ok